ui/ui-components/pages/model/[modelId]/source/[sourceId]/multipleProperties.tsx

```diff
@@ -130,7 +130,6 @@ export default function Page(props) {
         `/property`,
         {
           sourceId: source.id,
-          id: key, // reuse the key as ID
```

Comment:
@evantahler do you know what the reasoning for this was initially? (reusing they key as id when creating properties through the multiple properties page)

Reply:
Presumably we wanted to try to make pretty IDs (eg: user_id vs prop_abc123).  BUT, the config writer should take care of this for us... so I'm not that worried about removing this.